processor/src/stack/mod.rs

```diff
@@ -140,7 +140,10 @@ impl Stack {
         let mut stack_items = Vec::with_capacity(self.active_depth);
         self.trace.append_state_into(&mut stack_items, self.clk);
         self.overflow.append_into(&mut stack_items);
+        // TODO: Are we happy to expect() here. I think it makes sense as provided the processor
+        // logic is sound, the stack handling should be valid.
```

Comment:
Yep - I think `expect()` here is fine. If it fails, means there is a bug in the processor logic.